samples/bot-configuration-app/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-bot-configuration-app-nodejs",
+    "source": "officeDev",
+    "title": "Bot configuration app with type ahead search control",
+    "shortDescription": "This sample shows the feature of bot configuration and reconfiguration for bots in team and group chat.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-configuration-app/nodejs",
+    "longDescription": [
+      "This sample demonstrates the features of bot configuration and reconfiguration for both teams and group chats, including a type-ahead search (static and dynamic) control on Adaptive Cards."
+    ],
+    "creationDateTime": "2023-08-23",
```

Comment:
Use latest dates. in Readme file as well.

Reply:
Fixed